modules/core/client/components/ReadMorePanel.js

```diff
@@ -0,0 +1,71 @@
+// External dependencies
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+// Internal dependencies
+import { plainText, plainTextLength } from '../utils/filters';
+
+export class ReadMorePanel extends Component {
```

Comment:
Why a class component instead of a function?

Reply:
Just to be able to have `state.showMore `

Reply:
Doesn't the `useState` hook address that? It's used in the newer react components @mrkvon has done. Or am I missing something?

Reply:
Oh cool! Nah you're right, it's just me who hasn't learned React hooks yet and rolls like it's still 2019. 😁

Reply:
Updated in https://github.com/Trustroots/trustroots/pull/1161/commits/662494a65c2a0033e1b0f83e8b1079a186481104 Thanks for the heads up.